src/components/node/stanford-news/page-display.tsx

```diff
@@ -89,7 +89,7 @@ const StanfordNews = async ({node, ...props}: {node: NodeStanfordNews}) => {
 
       {imageUrl && (
         <figure className="relative mx-auto mb-100 aspect-[16/9] lg:w-10/12">
-          <Image className="object-cover" src={buildUrl(imageUrl).toString()} alt={imageAlt || ""} fill />
+          <Image className="!relative object-cover" src={buildUrl(imageUrl).toString()} alt={imageAlt || ""} fill />
```

Comment:
I would suggest staying away from `!important` uses, especially on the image component. NextJS Image component applies it's own styles that make it function the way it does for responsive sizes. My suggestion is to adjust the wrapper a little bit.:
```
<figure className="table">
  <span className="relative aspect-[16/9]">
     <Image .../>
  </span>
  <figcaption className="table-caption">...</figcaption>
</figure>
```
The `<Image>` component needs the immediate parent to be relative positioned. So changing this markup keeps that relative positioning but takes the caption out of it so the caption can be its own thing. You'll also notice the table classes. This can help with the positioning of the caption.

Reply:
@pookmish I used `caption` instead of `table-caption` because of how it displays with a table.